app/controllers/transition_landing_page_controller.rb

```diff
@@ -36,32 +34,11 @@ def presentable_section_items
   end
 
   def show_comms?
```

Comment:
does this do anything any more?

Reply:
I'm guessing the ability to turn it on and off was a requirement at some point? Has been there a while. Do you think I should remove it?

Reply:
Will merge to get the ab test off, and ask content peeps if it's still a thing.